cllc-public-app/Startup.cs

```diff
@@ -313,18 +313,16 @@ private void SetupServices(IServiceCollection services)
 
             // add BC Express Pay (Bambora) service
             services.AddHttpClient<IBCEPService, BCEPService>()
-                .AddPolicyHandler(GetRetryPolicy())
-                .AddPolicyHandler(GetCircuitBreakerPolicy());
```

Comment:
Quick question - why the removal of the circuit breaker here and for other services? Wouldn't that open up the API to infinite retries?

Reply:
We found in testing that the circuit breaker was being overly restrictive.  Once it was thrown it would not reset properly.  The retry policy has logic to limit the number of retries.